toxcore/TCP_connection_test.cc

```diff
@@ -0,0 +1,16 @@
+#include "tcp_connection.h"
+
+#include <gtest/gtest.h>
+
+namespace
+{
+
+// TODO(Jfreegman) make this useful or remove it after NGC is merged
+TEST(TCP_connection, NullTest)
+{
+    (void)tcp_connections_count(nullptr);
+    (void)tcp_connected_relays_count(nullptr);
+    (void)tcp_send_oob_packet_using_relay(nullptr, nullptr, nullptr, nullptr, 0);
```

Comment:
Don't pass any arguments.

`18:58 <@iphy> TEST(Foo, Bar) { (void)tcp_thing_oob_whatever; }`

I meant that missing `()`.